### ISSUE TITLE
Yelp app submission, please review. /cc @codepathreview

### ISSUE THREAD

**Issue by santos22** [santos22/Yelp#1]
Yelp app submission, please review. /cc @codepathreview

**Comment by minhchau273** [comment]
:+1: Great work! I do like the way you allow users to search restaurants. This is one of the most creative submissions I've ever seen :tada: :heart_eyes: 

This primary challenge of this homework was to learn how to use AutoLayout to build adaptive UI's that work across many different screen sizes and orientations. If you haven't already, we recommend going back and running your homework again while rotating the device / simulator to ensure your AutoLayout constraints function as expected.

### Some small comments:
- Note that you should resolve all the Auto Layout constraint warnings in your nibs.
- It's not a big deal but in case there is no result, you should show a label `No results found` to make your app more friendly :wink:
- For now, you only search within the current restaurants. For this assignment, you have to call the api to search the restaurants.
- `Implement map view of restaurant results` means you have to show the results as a map and pin the restaurant locations on that map.

We have a detailed [Project 3 Feedback Guide](http://courses.codepath.com/snippets/ios_university/feedback_guides/project_3_feedback.md) which covers the best practices for implementing this assignment. Read through the feedback guide point-by-point to determine ways you might be able to improve your submission. You should consider going back and implementing these improvements as well. Keep in mind that one of the most important parts of iOS development is learning the correct patterns and conventions.

If you have any particular questions about the assignment or the feedback, feel free to reply here or email us at universitysupport@codepath.com.

**Comment by santos22** [comment]
Hi @minhchau273, thanks for the prompt, valuable feedback!

I initially misunderstood the map view feature, but it is clear to me now; I will go ahead and make those changes to fix that :-)

In regards to the search feature, does it essentially do the same thing as my UIAlertController for when I typed in "donuts"? So let's say I typed in "thai" into the search bar, does that mean my table view would reload "thai" restaurant results? The user story requirements for the search bar weren't too specific, so that is why it only searches within the current restaurants.

Thanks!

**Comment by minhchau273** [comment]
Yes, that's right. When you type `thai` and search, the table view will reload `thai` results. Sorry for the unclear requirement. We will fix it.
